.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -233,6 +233,7 @@ swap:
   joblog: job log
   jobstream: job stream
   judgement: judgment
+  k8s: Kubernetes
```

Comment:
Wondering. Do we have legitimate instances of things like namespaces etc with k8s in the name? Maybe with dashes? Does this rule trigger in those instances?

Reply:
Update: When I removed the occurrences that were detected in auto-generated API docs, the results were 7 occurrences in 4 files.